poutyne/framework/experiment.py

```diff
@@ -879,3 +879,43 @@ def _test(self,
             test_stats.to_csv(self.test_log_filename.format(name=name), sep='\t', index=False)
 
         return ret
+
+    def is_better_than(self, another_experiment) -> bool:
+        """
+        Compare the results of the Experiment with another experiment. To compare, both Experiments need to be
+        logged, monitor the same metric and the same monitor mod ("min" or "max").
+
+        Args:
+            another_experiment (~poutyne. Experiment): Another Poutyne experiment to compare results with.
+
+        Return:
+            A bool if either or not the Experiment is better than the Experiment to compare with.
+        """
+        if not self.logging:
+            raise ValueError("The experiment is not logged.")
+        if not another_experiment.logging:
+            raise ValueError("The experiment to compare to is not logged.")
+
+        if self.monitor_metric != another_experiment.monitor_metric:
+            raise ValueError("The monitored metric is not the same between the two experiment.")
+        monitored_metric = self.monitor_metric
+
+        if self.monitor_mode != another_experiment.monitor_mode:
+            raise ValueError("The monitored mode is not the same between the two experiment.")
+        monitor_mode = self.monitor_mode
+
+        checkpoint = 'best' if self.monitoring else 'last'
+        self_stats = self.load_checkpoint(checkpoint, verbose=False)
+        self_monitored_metric = self_stats.get(monitored_metric)
+        self_monitored_metric_value = self_monitored_metric.to_numpy()[0]
```

Comment:
Might be a better way. Maybe something like

```python
self_monitored_metric = self_stats[monitored_metric].item()
```

Reply:
For testability, my approach is easier. Otherwise, the mocks are painful to set properly.

Reply:
switched to this approach